lsp-types/ChangeLog.md

```diff
@@ -1,5 +1,23 @@
 # Revision history for lsp-types
 
+## 1.4.0.0
+
+* Aeson 2 compatibility (#360) (@michaelpj)
+* Reduced dependency footprint (#383, #384) (@Bodigrim)
+* Use appropriate number types (#366) (@michaelpj)
+* Fix for #374 (#376) (@pepeiborra)
+* Fix virtual file name adding .hs extension (#364) (@heitor-lassarote, @jneira)
```

Comment:
```suggestion
* Fix virtual file name adding .hs extension (#364) (@heitor-lassarote)
```

I did not meningful change in that pr (afair :-P)

Reply:
You landed it!